### PR TITLE
Update Acceptence tests for data czar export key changes

### PR DESCRIPTION
**Pull request by macdiesel**

### CODE REVIEW

File: edx/analytics/tasks/tests/acceptance/test_database_export.py [hunk 1]
```diff
@@ -159,27 +161,35 @@ def write_exporter_config(self, org_id, course_id, config_file_path):
...
+        """)
+        org_config_text = org_config_text.format(
             environment=self.ENVIRONMENT,
-            sql_host=self.import_db.credentials['host'],
-            external_files=self.external_files_dir,
-            org_id=org_id,
             course_id=course_id,
         )
+        with open(org_config_file_path, 'w') as org_config_file:
+            org_config_file.write(config_text)
```
Comment (brianhw) [inline edx/analytics/tasks/tests/acceptance/test_database_export.py]: Should this be org_config_text?

File: edx/analytics/tasks/tests/acceptance/test_database_export.py [hunk 2]
```diff
@@ -159,27 +161,35 @@ def write_exporter_config(self, org_id, course_id, config_file_path):
...
                 external_files: {external_files}
+            """)
+        config_text = env_config_text.format(
+            sql_user=self.import_db.credentials['username'],
+            sql_db=self.import_db.database_name,
+            sql_password=self.import_db.credentials['password'],
+            sql_host=self.import_db.credentials['host'],
+            external_files=self.external_files_dir,
+            org_id=org_id,
+
```
Comment (brianhw) [inline edx/analytics/tasks/tests/acceptance/test_database_export.py]: Is this missing environment=self.ENVIRONMENT that was used before?

**Review by brianhw — COMMENTED** [review]
Few questions.

File: edx/analytics/tasks/tests/acceptance/test_database_export.py [hunk 3]
```diff
@@ -159,27 +161,35 @@ def write_exporter_config(self, org_id, course_id, config_file_path):
                 name: {environment}-analytics
                 sql_host: {sql_host}
                 external_files: {external_files}
+            """)
+        config_text = env_config_text.format(
```
Comment (iloveagent57) [inline edx/analytics/tasks/tests/acceptance/test_database_export.py]: Maybe better to call the `env_config_text`.

File: edx/analytics/tasks/tests/acceptance/test_database_export.py [hunk 4]
```diff
@@ -159,27 +161,35 @@ def write_exporter_config(self, org_id, course_id, config_file_path):
...
+            sql_user=self.import_db.credentials['username'],
+            sql_db=self.import_db.database_name,
+            sql_password=self.import_db.credentials['password'],
+            sql_host=self.import_db.credentials['host'],
+            external_files=self.external_files_dir,
+            org_id=org_id,
+
+        )
+        with open(env_config_file_path, 'w') as env_config_file:
+            env_config_file.write(config_text)
```
Comment (iloveagent57) [inline edx/analytics/tasks/tests/acceptance/test_database_export.py]: ...and change it here.

File: edx/analytics/tasks/tests/acceptance/test_database_export.py (hunk 1 above)
Comment (iloveagent57) [reply edx/analytics/tasks/tests/acceptance/test_database_export.py]: I think it should be.

**Review by iloveagent57 — CHANGES_REQUESTED** [review]
Looks good, other than the `org_config_text` thing that Brian pointed out, which I think will make this do the wrong thing.

**Comment by iloveagent57** [comment]
...err, well, I guess I was looking at a test file, so "doing the wrong thing" may be over the top, but you get the idea...

**Comment by macdiesel** [comment]
My apologies, this was sloppy, won't happen again.

Ready for re-review.

File: edx/analytics/tasks/tests/acceptance/test_database_export.py [hunk 5]
```diff
@@ -123,9 +123,10 @@ def run_legacy_exporter(self, org_id, course_id):
         Uploads the package to s3://<exporter_output_bucket>/<output_prefix>edx-<year>-<month>-<day>.zip
 
         """
-        config_file_path = os.path.join(self.temporary_dir, '{}_acceptance.yml'.format(org_id))
+        env_config_file_path = os.path.join(self.temporary_dir, '{}_env_acceptance.yml'.format(org_id))
+        org_config_file_path = os.path.join(self.temporary_dir, '{}_org_acceptance.yml'.format(org_id))
 
-        self.write_exporter_config(org_id, course_id, config_file_path)
+        self.write_exporter_config(org_id, course_id, env_config_file_path)
```
Comment (brianhw) [inline edx/analytics/tasks/tests/acceptance/test_database_export.py]: Isn't this supposed to have two config files?

**Review by brianhw — COMMENTED** [review]
Another question.

**Comment by brianhw** [comment]
I assume this PR should be included as a sub-task of DE-414.   The acceptance tests in master are still breaking for test_database_export.ExportAcceptanceTest.test_database_export.

**Comment by macdiesel** [comment]
Codecov says these lines are not exercised during the test. Is s3 available in travis to run these tests? If not I can mark these areas as # pragma: no cover and merge.

**Review by azafty468 — APPROVED** [review]
👍

**Comment by azafty468** [comment]
I do not belief S3 is available for Travis but I'm not 100% positive

**Pull request merged**